pkg/kubelet/kubelet_server_journal_windows.go

```diff
@@ -0,0 +1,86 @@
+//go:build windows
+
+/*
+Copyright 2022 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package kubelet
+
+import (
+	"context"
+	"fmt"
+	"os/exec"
+	"strings"
+)
+
+const powershellExe = "PowerShell.exe"
+
+// getLoggingCmd returns the powershell cmd and arguments for the given nodeLogQuery and boot
+func getLoggingCmd(n *nodeLogQuery, services []string) (string, []string, error) {
+	args := []string{
+		"-NonInteractive",
+		"-ExecutionPolicy", "Bypass",
+		"-Command",
+	}
+
+	psCmd := "Get-WinEvent -FilterHashtable @{LogName='Application'"
+	if n.SinceTime != nil {
+		psCmd += fmt.Sprintf("; StartTime='%s'", strings.TrimSuffix(n.SinceTime.UTC().String(), " +0000 UTC"))
```

Comment:
same comment about formatting time directly instead of calling String() and trimming